src/metrics/components/MetricsLayout/MetricsLayout.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { render, waitForI18n } from '../../../test-utils';
+import { MetricsLayout } from '../MetricsLayout/MetricsLayout';
+jest.mock('../../utils', () => {
```

Comment:
Why do we need to mock `utils` ?

Reply:
Actually instead of using real implementation of utils i thought to mock the utils folder so that it not uses actual implementation of utils.

Reply:
if we din't used mocked utils then also it will executes the tests according to our conditions in the component. for that we need to directly import the MetricsType from utils directly in our test file.